.github/workflows/ci.yml

```diff
@@ -69,6 +69,11 @@ jobs:
           cd ${GITHUB_WORKSPACE}/src/github.com/mattermost/mattermost-operator
           make unittest goverall
 
+      - name: Install kind tooling
```

Comment:
I think this is may redundant right? we already try to download kind here and then run it https://github.com/mattermost/mattermost-operator/blob/master/test/e2e.sh#L25.